### ISSUE TITLE
javascript error visiting an edit page with tags "SyntaxError: missing ] after element list"

### ISSUE THREAD

**Issue by davidemoro** [Kotti/Kotti#562]
Preconditions:
* a document (or any other entity) with a tab

How to reproduce:
* login
* visit our target resource edit form

Expected result:
* no javascript errors
* tags widget displayed as usual (list of tags visually presented with x icon)

Actual result:
* javascript error "SyntaxError: missing ] after element list"
* broken UI for tag widget

Workaround:
* not a blocking issue because you are able to manage tags editing a comma separated string representing tags

Javascript producing error:
```
  deform.addCallback(
    'deformField3',
    function(oid) {
      $('#' + oid).tagit({
          availableTags: [b'login'],
          allowSpaces: true
      });
    }
  )
```

requirements.txt:
```
alembic==1.0.11
appdirs==1.4.3
Babel==2.7.0
Beaker==1.10.1
bleach==3.1.0
bleach-whitelist==0.0.10
Chameleon==3.6.2
colander==1.7.0
deform==2.0.7
docopt==0.6.2
fanstatic==1.1
filedepot==0.6.0
FormEncode==2.0.0a1
html2text==2018.1.9
hupper==1.8.1
iso8601==0.1.11
js.angular==1.1.4
js.bootstrap==3.3.4
js.deform==2.0.3
js.fineuploader==5.14.0
js.html5shiv==3.7.3
js.jquery==1.9.1
js.jquery-form==3.9
js.jquery-maskedinput==1.4.1
js.jquery-maskmoney==3.0.2
js.jquery-sortable==0.9.13
js.jquery-tablednd==0.4
js.jquery-timepicker-addon==1.5.3
js.jqueryui==1.10.3
js.jqueryui-tagit==2.0.24.post2
js.modernizr==2.5.3.1
js.select2==4.0.4.post1
js.tinymce==4.5.4
Kotti==2.0.1
LEPL==5.1.3
lingua==4.13
Mako==1.0.12
MarkupSafe==1.1.1
PasteDeploy==2.0.1
peppercorn==0.6
pkg-resources==0.0.0
plaster==1.0
plaster-pastedeploy==0.7
polib==1.1.0
py-bcrypt==0.4
pyramid==1.10.4
pyramid-beaker==0.8
pyramid-chameleon==0.3
pyramid-deform==0.2
pyramid-mailer==0.15.1
pyramid-tm==2.2.1
pyramid-zcml==1.2.0
python-dateutil==2.8.0
python-editor==1.0.4
pytz==2019.1
repoze.lru==0.7
repoze.sendmail==4.4.1
repoze.workflow==1.0b1
repoze.zcml==1.0b1
rfc6266-parser==0.0.5.post2
shutilwhich==1.1.0
six==1.12.0
SQLAlchemy==1.3.5
SQLAlchemy-Utils==0.34.0
transaction==2.4.0
translationstring==1.3
Unidecode==1.1.1
usersettings==1.0.7
venusian==1.2.0
waitress==1.3.0
webencodings==0.5.1
WebOb==1.8.5
zope.component==4.5
zope.configuration==4.3.1
zope.deferredimport==4.3
zope.deprecation==4.4.0
zope.event==4.4
zope.hookable==4.2.0
zope.i18nmessageid==4.3.1
zope.interface==4.6.0
zope.proxy==4.3.1
zope.schema==4.9.3
zope.sqlalchemy==1.1

```

**Comment by disko** [comment]
``b'login'`` looks like a serialisation issue introduced with the Python 3 migration. I'll look into it ASAP.

**Comment by disko** [comment]
I guess https://github.com/Kotti/Kotti/blob/master/kotti/templates/deform/tag_it.pt#L8 needs to be replaced with sth. like ``[str(t) for t in field.widget.available_tags]``

**Comment by disko** [comment]
Fixed in 1a231728edacda9f2d061d853a3023dbfb58bc32. Thanks for reporting!

**Comment by davidemoro** [comment]
Thank you very much @disko!
